new animax source

jlapitan opened the pull request.

jlapitan committed:
add bein sports hd for Qatar and UAE

jlapitan committed:
Add animax for korea

jlapitan committed:
remove animax in kr.meu, remove bein sports in ae.m3u, ref #487

jlapitan committed:
Add new animax

jlapitan committed:
add fox life channel

boomski commented on channels/kr.m3u:
```diff
@@ -165,3 +165,5 @@ http://54.180.143.6:9981/stream/channelid/750850617
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/r1WTYTa.png" group-title="",현대홈쇼핑+샵 / + Shop
 http://54.180.143.6:9981/stream/channelid/965307250
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://imgur.com/2TSc2pJ.png" group-title="",Animax
+http://211.49.191.148:9981/stream/channelid/1158483964
```
![afbeelding](https://user-images.githubusercontent.com/25525836/66757918-7adb9d80-ee9d-11e9-9478-75c7880f8969.png)
Link does not work

jlapitan replied:
<img width="892" alt="Screen Shot 2019-10-15 at 1 43 28 AM" src="https://user-images.githubusercontent.com/1114030/66785105-4b548180-eeed-11e9-9385-1623a740097e.png">

freearhey replied:
stream works for me. with lags but works (tested with VLC and streamtest)

boomski commented on channels/kr.m3u:
```diff
@@ -167,3 +167,5 @@ http://54.180.143.6:9981/stream/channelid/1805607194
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://imgur.com/2TSc2pJ.png" group-title="",Animax
 http://211.49.191.148:9981/stream/channelid/1158483964
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/WRzx3F9.png" group-title="",Fox Life
+http://211.49.191.148:9981/stream/channelid/1960414728
```
![afbeelding](https://user-images.githubusercontent.com/25525836/66758023-bd04df00-ee9d-11e9-9b62-e6053f68475e.png)
Link does not work

jlapitan replied:
<img width="837" alt="Screen Shot 2019-10-15 at 1 45 43 AM" src="https://user-images.githubusercontent.com/1114030/66785179-89ea3c00-eeed-11e9-857c-0f81969b9ad7.png">
link is working

freearhey replied:
same thing, stream works but it is not possible to watch due to lags

boomski commented:
I still getting error on these links  , maybe geoblocked?

jlapitan commented:
> I still getting error on these links , maybe geoblocked?
Not sure but these are the results test link

https://streamtest.in/logs/http-211-49-191-148-9981-stream-channelid-1960414728-696b17ef-dfa1-4465-a69c-4cdc8f701ade

https://streamtest.in/logs/http-211-49-191-148-9981-stream-channelid-1158483964-3a125a72-cdd9-450a-ab72-d2bfa100019e

jlapitan committed:
Update ae.m3u

add Dubai one channel

test:

https://streamtest.in/logs/http-barakatiptv-com-80-live-jc65xhkf5j-gg1rnsdngf-69952-m3u8

freearhey commented on channels/qa.m3u:
```diff
@@ -17,3 +17,6 @@ https://alrayyan1live.azureedge.net/alrayyan1/alrayyan1.smil/chunklist_b1428000.
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
 http://95.170.215.101/hls/m3u8/Bein-Sport-HD1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
+http://line.protv.cc:8000/live/tNxE4rdouh/ttqs53nnsu/7776.m3u8
```
currently not working (tested with VLC and streamtest)

jlapitan replied:
non-working stream has been removed

freearhey reviewed (changes requested):
Please remove all broken links from PR so that we can accept it.

jlapitan committed:
Update qa.m3u

remove non-working stream
